src/v2/ion/uiSchemaLabelTransformer.js

```diff
@@ -0,0 +1,143 @@
+/*!
+ * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
```

Comment:
is this comment necessary in this file ?

Reply:
I think we shall add comments to all JS files.